src/panel/tableStore.spec.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { TableStore, RowGroup } from './tableStore';
+import { observable } from 'mobx';
+import { _Region } from '../shared';
+import assert from 'assert';
+
+describe('TableStore', () => {
+    it('Get rows based on different values of expanded property', () => {
+        // Create instance of tableStore
+        const groupBy = (item: number) => item % 2 == 0 ? 'even' : 'odd';
+        const itemSource = { results: [1,2,3,4,5] }
+        const selection = observable.box(); 
+        const tableStore = new TableStore(groupBy, itemSource, selection);
```

Comment:
We should find a way to not copy/paste lines 12-15 in this file.

Reply:
Moved `groupBy` and `selection` to outside the `it` blocks but inside `describe` block. I can probably create a utility method which creates a tableStore instance based on `results` provided (since thats the only thing changing between tests). Any suggestion?